dockerfiles/hpcc/templates/dllserver.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
 spec:
   storageClassName: manual
   capacity:
-    storage: 10Gi
+    storage: {{ .Values.global.dllserver.storageSize }}
```

Comment:
@richardkchapman - not directly related - but fixed.

Reply:
Not 100% sure this is right. There's a subtle distinction between the volume size when setting up a local mount and the size of a pv request. Do we always want them to match?